Tools/README

```diff
@@ -19,7 +19,7 @@ iobench         Benchmark for the new Python I/O system. (*)
 
 msi             Support for packaging Python as an MSI package on Windows.
 
-parser          Un-parsing tool to generate code from an AST.
+wasm            help facilitate cross compilation of CPython to WebAssembly (WASM)
```

Comment:
Based on `Tools/wasm/README.md`:
```suggestion
wasm            Config and helpers to facilitate cross compilation of CPython
                to WebAssembly (WASM).
```
And please keep these in alphabetical order.